next-i18next/trigger.sh

```diff
@@ -17,7 +17,7 @@ echo -e "${GREEN}[Step 1] Installing additional packages${NC}"
 echo ""
 echo -e "Packages: ${GREEN}next-i18next"
```

Comment:
hi thanks for the PR, one more thing can you also add them here?

```suggestion
echo -e "Packages: ${GREEN}next-i18next i18next react-i18next"
```